### ISSUE TITLE
How to connect to mupx mongodb externally

### ISSUE THREAD

**Issue by avishaan** [arunoda/meteor-up-legacy#1106]
I use to be able to access my mongo database in mup using the following http://blog.codehatcher.com/mongodb-ssh-access-on-remote-meteor-deployment and it made a lot of the maintenance, backup, restoring, and testing very convenient. Basically, SSH in then connect to the localhost.

I'm trying to do something similar with mupx but can't figure out what I am missing, could use a little help with this. Thanks in advance from a long time user and fan

**Comment by brmk** [comment]
I don't think that you can do this since mupx is using a docker container to host your application.
You can only access your mongodb if you login to your server and execute
`docker exec -it mongodb mongo <appName>`

Read this here:
https://github.com/arunoda/meteor-up/tree/mupx#accessing-the-database

**Comment by avishaan** [comment]
Kind of a shame but really appreciate the follow up. If someone comes up with a clever solution to this, let me know

Sent using CloudMagic Email [https://cloudmagic.com/k/d/mailapp?ct=pi&cv=7.6.23&pv=9.2.1&source=email_footer_2]
On Mon, Aug 8, 2016 at 11:07 AM, Ihor Brmk notifications@github.com wrote:
I don't think that you can do this since mupx is using a docker container to host your application.
You can only access your mongodb if you login to your server and execute
docker exec -it mongodb mongo <appName>

Read this here:
https://github.com/arunoda/meteor-up/tree/mupx#accessing-the-database [https://github.com/arunoda/meteor-up/tree/mupx#accessing-the-database]
